### PR TITLE
Fix acceptance test fixtures

### PR DESCRIPTION
**Pull request by bmedx**
This PR fixes downstream fixtures broken by the addition of new data in https://github.com/edx/edx-analytics-pipeline/pull/770 . The acceptance tests have been broken since it merged.

### CODE REVIEW

File: edx/analytics/tasks/tests/acceptance/fixtures/input/courses.json [hunk 1]
```diff
@@ -1,333 +1,2 @@
...
...{      "name": "Testing",      "subtitle": "Learn about testing.",      "description": "EdX offers a wide variety of free online testing courses.",      "banner_image_url": "https://www.edx.org/sites/default/files/testing-1440x210.jpg",      "card_image_url": "https://www.edx.org/sites/default/files/subject/image/card/testing.jpg",      "slug": "testing",      "uuid": "74b6ed2a-3ba0-49be-adc9-53f7256a12e1"    }  ],  "prerequisites": [],  "prerequisites_raw": "None",  "expected_learning_items": [],  "video": null,  "sponsors": [],  "modified": "2017-11-30T20:44:59.285976Z",  "marketing_url": "https://www.edx.org/course/all-about-acceptance-testing-edx-testing101",  "syllabus_raw": "None",  "outcome": "<ul><li>Why Acceptance Testing still matters, all through a focus on testing</li>\n\t<li>Ideas and techniques that will help you understand as well as enjoy acceptance testing</li>\n\t<li>How to understand and appreciate acceptance testing and its power</li>\n</ul>",  "programs": [    {      "uuid": "acb243a0-1234-5abe-099e-ffcae2a340d4",      "title": "Testing",      "type": "XSeries",      "marketing_slug": "tests",      "marketing_url": "https://example.com/xseries/testing"    }  ]}
```
Comment (bmedx) [inline edx/analytics/tasks/tests/acceptance/fixtures/input/courses.json:1]: This change is just squashing the json back onto one line, which is necessary for it to load successfully here: https://github.com/edx/edx-analytics-pipeline/blob/master/edx/analytics/tasks/warehouse/load_internal_reporting_course_catalog.py#L208

**Review by HassanJaveed84 — APPROVED** [review]
LGTM. Do we want to run the full acceptance test suite with these changes ?

File: edx/analytics/tasks/tests/acceptance/__init__.py [hunk 2]
```diff
@@ -373,11 +373,19 @@ def assert_data_frames_equal(data, expected):
         try:
             assert_frame_equal(data, expected)
         except AssertionError:
-            pandas.set_option('display.max_columns', None)
+            # For some reason the version of pands we have pinned throws an error if you try to print it
```
Comment (doctoryes) [inline edx/analytics/tasks/tests/acceptance/__init__.py:376]: Nit: `pands` -> `pandas`

**Review by doctoryes — APPROVED** [review]

**Review by brianhw — APPROVED** [review]
LGTM.  👍

**Comment by bmedx** [comment]
> LGTM. Do we want to run the full acceptance test suite with these changes ?

Full acceptance tests ran and passed. 👍

**Pull request merged**